core/trino-main/src/main/java/io/trino/metadata/SessionPropertyManager.java

```diff
@@ -209,9 +209,9 @@ public void validateSystemSessionProperty(String propertyName, String propertyVa
         decodePropertyValue(propertyName, propertyValue, propertyMetadata.getJavaType(), propertyMetadata);
     }
 
-    public void validateCatalogSessionProperty(CatalogName catalog, String catalogName, String propertyName, String propertyValue)
+    public void validateCatalogSessionProperty(CatalogName catalog, String propertyName, String propertyValue)
```

Comment:
there were no call spots?

Reply:
Also I think it may have a purpose (or maybe just used to have).
@findepi do you know maybe why we often use `String` and `CatalogName` aside together?

Reply:
i do not. maybe just a remnant from before https://github.com/trinodb/trino/pull/406? cc @kokosing

Reply:
I haven't found more locations, but yes I would like to here @findepi opinion. It's easy to drop this commit.

Reply:
> I haven't found more locations

I mean this commit changes method signature but does not change any call-spots. This seems wrong (unless method is not needed at all). I think you are actually changing call-spots in next commit.

Reply:
🤦‍♂️ yes my bad. IT seems my yesterday rebasing made some mess.
Fixed now.